.github/workflows/ci-comment-failures.yml

```
@@ -178,3 +178,51 @@ jobs:
               issue_number: issue_number,
               body: 'Your PR increases Bevy Minimum Supported Rust Version. Please update the `rust-version` field in the root Cargo.toml file.'
             });
+
+  make-macos-screenshots-available:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - name: 'Download artifact'
+        id: find-artifact
+        uses: actions/github-script@v7
+        with:
+          result-encoding: string
+          script: |
+            var artifacts = await github.rest.actions.listWorkflowRunArtifacts({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              run_id: ${{github.event.workflow_run.id }},
```

```suggestion
              run_id: ${{ github.event.workflow_run.id }},
```

(Nit) Add a space to make it symmetric.